subxt/src/blocks/extrinsic_types.rs

```diff
@@ -420,18 +420,16 @@ where
 
     /// Decode and provide the extrinsic fields back in the form of a [`scale_value::Composite`]
     /// type which represents the named or unnamed fields that were present in the extrinsic.
-    pub fn field_values(
-        &self,
-    ) -> Result<scale_value::Composite<scale_value::scale::TypeId>, Error> {
```

Comment:
Note: previously we were able to return `scale_value::Composite<u32>`, but that would require changes to `scale_value`, e.g. creating a new fn `decode_composite_as_fields`, that maps the type id correctly.

Reply:
I don't think that this is a deal breaker, but I also think that exposing a fewwstanding `decode_as_fields` in `scale-value` alongside `decode_as_type` would be good to have (and should be a patch bump release too :)). That way, `scale-value` provides all of the same capabilityies as before (and internally its `DecodeAsFields` impl will just use this freestanding function too)